pkg/logs/internal/tailers/tailer_tracker_test.go

```diff
@@ -0,0 +1,97 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package tailers
+
+import (
+	"testing"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/internal/status"
+	assert "github.com/stretchr/testify/require"
+)
+
+type TestTailer1 struct {
+	id   string
+	info *status.InfoRegistry
+}
+
+func NewTestTaler1(id string) *TestTailer1 {
+	return &TestTailer1{
+		id:   id,
+		info: status.NewInfoRegistry(),
+	}
+}
+
+func (t *TestTailer1) GetId() string {
+	return t.id
+}
+func (t *TestTailer1) GetType() string {
+	return "test"
+}
+func (t *TestTailer1) GetInfo() *status.InfoRegistry {
+	return t.info
+}
+
+type TestTailer2 struct {
+	id   string
+	info *status.InfoRegistry
+}
+
+func NewTestTaler2(id string) *TestTailer2 {
```

Comment:
```suggestion
func NewTestTailer2(id string) *TestTailer2 {
```

Reply:
🤦 (same for `NewTestTailer1`). Will fix.